src/raft.rs

```diff
@@ -2458,7 +2497,7 @@ impl<T: Storage> Raft<T> {
         } else {
             debug!(
                 self.logger,
-                "rejected msgApp [logterm: {msg_log_term}, index: {msg_index}] \
+                "Reject append [logterm: {msg_log_term}, index: {msg_index}] \
```

Comment:
use lowercase for the initial letter, same for other places to fit the log style

Reply:
Resolved.